Cargo.toml

```
@@ -35,6 +35,7 @@ no_std = [
 std = [
   "fontdb/memmap",
   "fontdb/std",
+  "ouroboros/std",
```

If ouroboros is becoming optional based on features, this should be made the following instead to ensure enabling std does not needlessly pull in ouroboros

```suggestion
  "ouroboros?/std",
```

If that's not intended, then it should probably be noted since I am a bit confused if that is the case.

---

Ouroboros is not becoming optional, but rather becoming mandatory. Previously, ouroboros was only used for `FontSystem` if the `std` feature was used. Now it will be always used for `Font`. Note that I didn't change the ouroboros entry to be optional, I just disabled the default-features (`std`) and enabled it only if the `std` feature is used.

The current main is wrong with not disabling the `std` feature of ouroboros, but it didn't lead to issues because the ouroboros macro was only used when the `std` feature was active. I hope this makes sense.

---

Oh I guess I misread default-features as a optional...